package.json

```diff
@@ -37,8 +38,19 @@
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-1": "^6.16.0",
     "babel-runtime": "^6.18.0",
+    "chai": "^3.5.0",
+    "chai-as-promised": "^6.0.0",
+    "chai-enzyme": "^0.6.0",
+    "dirty-chai": "^1.2.2",
+    "enzyme": "^2.6.0",
     "eslint": "^3.8.1",
     "eslint-config-ta": "^5.2.0",
+    "jsdom": "^9.8.3",
```

Comment:
Would prefer to use Karma so this is consistent with our other stacks.  It also means it will port to Genesis later.  If there is any push back, ship as is.